larky/src/main/resources/vendor/ISO8583/Parser/Parser.star

```diff
@@ -0,0 +1,13 @@
+load("@stdlib//larky", larky="larky")
+load("@stdlib//jiso8583", _JISO8583="jiso8583")
+
+decode = _JISO8583.Parser.decode
+
+def _decode(bytes, packager_xml_string):
```

Comment:
as args are a part of public api, let's name them accordingly
in [pyison8583](https://github.com/knovichikhin/pyiso8583) second param is `spec` for `specification`.
let's do the same

Reply:
done